frontend/src/views/JoinAGameView.vue

```diff
@@ -34,34 +35,87 @@ import type IGameSettings from '@/interfaces/game/IGameSettings'
 const router = useRouter()
 
 const gameStore = useGameStore();
+
+//const currentGame = ref<IGameSettings | null>(gameStore.currentGame)
+
 const isLoading = ref(false);
 const useClassicMode = ref(true);
 const queueTimer = ref(0)
 let queueTimerIntervalID = 0
 
+function inviteToPlay() : boolean {
+	console.log("invite !")
+	gameStore.currentGame.game.type = useClassicMode.value ? 'CLASSIC' : 'CUSTOM'
+	const invite = {
+		gameType: gameStore.currentGame.game.type,
+		userId: gameStore.currentGame.player2.id,
+	}
+	gameStore.socket.emit("invitePlayer", invite, ((res: string) => {
+		if (res !== "OK") {
+			cancelInvitation()
+			window.alert(res)
```

Comment:
also waiting for modal ?

Reply:
Not sure what you mean here. The alert is synchronous and blocks the whole page, so yes it is waiting for the alert to be closed

Reply:
This would need to be changed if @YvanoffP implements a custom modal to replace the `window.alert`, `window.prompt` and `window.dialog`